docs/howto/team-open-science-mindset.md

```diff
@@ -21,46 +21,77 @@ We want our community ontology editors and developers to be _excited_ to be part
 
 1. **Take ownership of our community ontologies**
 
-- Ensure that you see your issues and Pull Requests through to the end
-- Feel empowered to nag reviewers or experts to help
+    - Ensure that you see your issues and Pull Requests through to the end
+    - Feel empowered to nag reviewers or experts to help
+
+Example: After waiting for the PR to be reviewed, Meghan kindly asked Nicole if she should find a different reviewer.
+
+<img width="787" alt="image" src="https://user-images.githubusercontent.com/6722114/190458254-ed13ef7b-9c0a-455a-ad49-89d2548b81ab.png">
+
 
 1. **Reduce the fear of breaking the ontology**
 
-- We've all probably already made the same mistakes before - it's okay to break things or or make mistakes..
-- We have so many checks in place and GitHub has version control. Nothing has ever been broken to the point where it can't be fixed.
-- Perfect is the enemy of good enough.
-- It's okay if your PR is rejected
+    - We've all probably already made the same mistakes before - it's okay to break things or or make mistakes.
+
+Example: The QC checks on [this PR](https://github.com/monarch-initiative/mondo/pull/5266) failed 6 times before it passed.
+
```

Comment:
It looks like the checks on this PR are still in a failed state.

Reply:
I agree, but I am not concerned now.